### ISSUE TITLE
CI: Automatic hub image scanning using trivy

### ISSUE THREAD

**Issue by wongannaw** [jupyterhub/configurable-http-proxy#269]
## Bug description
Running trivy image scanning against the image for http-configurable-proxy results in two vulnerabilities found.

## Expected behavior
0 vulnerabilities

## Actual behavior
`trivy image --ignore-unfixed --light jupyterhub/configurable-http-proxy:latest > out.txt`

Results in: out.txt showing 2 vulnerabilities

## How to reproduce

1. Install trivy
2. Run `docker pull jupyterhub/configurable-http-proxy:latest`
3. Run `trivy image --ignore-unfixed jupyterhub/configurable-http-proxy:latest > out.txt`

## Your personal set up

- OS: MacOS
- Version:  jupyterhub/configurable-http-proxy:latest
- Configuration: No configuration

**Comment by wongannaw** [comment]
Adding `out.txt` output below:

```
2020-10-19T10:18:40.356-0500	[33mWARN[0m	You should avoid using the :latest tag as it is cached. You need to specify '--clear-cache' option when :latest image is changed
2020-10-19T10:18:40.393-0500	[34mINFO[0m	Detecting Alpine vulnerabilities...
2020-10-19T10:18:40.394-0500	[34mINFO[0m	Detecting nodejs vulnerabilities...

jupyterhub/configurable-http-proxy:latest (alpine 3.11.6)
=========================================================
Total: 2 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 2, CRITICAL: 0)

+-----------+------------------+----------+-------------------+---------------+
|  LIBRARY  | VULNERABILITY ID | SEVERITY | INSTALLED VERSION | FIXED VERSION |
+-----------+------------------+----------+-------------------+---------------+
| libgcc    | CVE-2019-15847   | HIGH     | 9.2.0-r4          | 9.3.0-r0      |
+-----------+                  +          +                   +               +
| libstdc++ |                  |          |                   |               |
+-----------+------------------+----------+-------------------+---------------+

srv/configurable-http-proxy/package-lock.json
=============================================
Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)


```

**Comment by wongannaw** [comment]
Fixed by #270

**Comment by consideRatio** [comment]
The title isnt reflecting the issue though!
